packages/config/src/bin/serve-chrome.ts

```diff
@@ -0,0 +1,164 @@
+import axios from 'axios';
+import path from 'path';
+import fs from 'fs';
+import { execSync, spawn } from 'child_process';
+import { load } from 'js-yaml';
+import { LogType, fecLogger } from '@redhat-cloud-services/frontend-components-config-utilities';
+import waitOn from 'wait-on';
+
+const REPO_OWNER = 'RedHatInsights';
+const REPO_NAME = 'insights-chrome';
+const CONTAINER_PORT = 8000;
+export const EXPOSED_PORT = 9999;
+
+type ContainerRuntime = 'docker' | 'podman';
+let execBin: ContainerRuntime | undefined = undefined;
+
+const chromeDeploymentConfig = {
+  repo: 'git@gitlab.cee.redhat.com:service/app-interface.git',
```

Comment:
This is not a public repo.

Reply:
It is not. And we will fallback to the GH if for some reason it is available. It is public as ling as you are on the company network.

Reply:
@Hyperkid123, will that affect HCC **app** developers' experience?

As a dev' on the OCM UI (@ `/openshift`), currently I don't have to be connected to company network (i.e. be logged in to VPN) in order to run my local dev-server.

Reply:
@eliranmal  Then you are not proxying chrome and you are pulling requesting assets directly from production CDN. If you are using the shared config (or just the proxy) you are not using production assets, but a dev build from the build repository in GH. So you would in this case fall back to the GitHub latest commit. Which is identical to what you are probably using today.

Reply:
Thanx for the elaboration, @Hyperkid123 !

We're just in the process of containerizing our deployment builds, and as a by-produce, might change the way we build to not use that (intermediate) build repo.
Would we have to be logged-in to VPN after that change?

Maybe a broader question then; under which scenarios will VPN be required?

Reply:
If you want to develop in the stage environment. The production uses VPN by default but the requirement can be disabled in the current setup.

Reply:
great, thanx!